client/landing/stepper/declarative-flow/internals/steps-repository/readymade-template-generate-content/index.tsx

```diff
@@ -1,23 +1,139 @@
-import { __experimentalNavigatorProvider as NavigatorProvider } from '@wordpress/components';
-import { useInitialPath } from 'calypso/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/hooks';
-import type { Step } from '../../types';
+import { FormLabel } from '@automattic/components';
+import { StepContainer } from '@automattic/onboarding';
+import { Button } from '@wordpress/components';
+import { useDispatch, useSelect } from '@wordpress/data';
+import { translate } from 'i18n-calypso';
+import React, { useState } from 'react';
+import useUrlQueryParam from 'calypso/a8c-for-agencies/hooks/use-url-query-param';
+import DocumentHead from 'calypso/components/data/document-head';
+import FormattedHeader from 'calypso/components/formatted-header';
+import FormFieldset from 'calypso/components/forms/form-fieldset';
+import FormTextarea from 'calypso/components/forms/form-textarea';
+import { Step } from 'calypso/landing/stepper/declarative-flow/internals/types';
+import { ONBOARD_STORE, SITE_STORE } from 'calypso/landing/stepper/stores';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import { ReadymadeTemplate } from 'calypso/my-sites/patterns/types';
+import generateAIContentForTemplate from './api/generate-content';
+import ReadymadeTemplatePreview from './components/readymade-template-preview';
+import TextProgressBar from './components/text-progress-bar';
+import type { OnboardSelect } from '@automattic/data-stores';
+import './style.scss';
+
+type ReadymadeTemplateGenerateContentProps = {
+	readymadeTemplate: ReadymadeTemplate;
+	siteId: number;
+	siteSlug: string;
+};
+
+const ReadymadeTemplateGenerateContent: React.FC< ReadymadeTemplateGenerateContentProps > = ( {
+	readymadeTemplate,
+	siteId,
+	siteSlug,
+} ) => {
+	const defaultContext = translate( 'Write an amazing description of your site.' );
```

Comment:
I think this should be displayed as a placeholder in the textarea instead of as an actual value:

Current | Expected
--- | ---
<img width="398" alt="Screenshot 2024-08-02 at 13 32 27" src="https://github.com/user-attachments/assets/79601ed0-f00e-4d1d-ad08-78420d741dc4"> | <img width="421" alt="Screenshot 2024-08-02 at 13 34 30" src="https://github.com/user-attachments/assets/2a8b5c25-5c23-4ad8-bd52-782b877dd061">

Reply:
this will make the red blob clearer too, as it won't appear until you've started typing something.

Reply:
Done